macOS_Catalina.md

```diff
@@ -17,14 +17,13 @@ If `ProductVersion` is less then `10.15` then this document is not for you. Norm
 
 ### The acid test
 To see if `Xcode Command Line Tools` is installed in a way that will work with `node-gyp`, run:
-1. `/usr/sbin/pkgutil --packages | grep CL`
-    * `com.apple.pkg.CLTools_Executables` should be listed. If it isn't, this test failed.
-2. `/usr/sbin/pkgutil --pkg-info com.apple.pkg.CLTools_Executables`
-    * `version: 11.0.0` (or later) should be listed. If it isn't, this test failed.
-    
-If both tests succeeded, _you are done_! You should be ready to install `node-gyp`.
+```
+curl -L https://git.io/node-gyp:acid-test | bash
```

Comment:
I'm not entirely comfortable promoting a shortened link, piping anything to bash involves a lot of trust so I'd rather this be explicit _and_ also present a more manual way of doing it. You could update your text to point directly to `acid-test.sh` if they want to download it themselves, and then use the full URL as an option if they're comfortable: `https://github.com/nodejs/node-gyp/blob/master/acid-test.sh`

_But_ also this test needs to be renamed, I know this makes it tedious, but this is only an issue with macOS Catalina and we need to separate this out as a concern. I suggest: `macOS_Catalina_acid_test.sh` so it shows up right next to this doc.

I'm not vouching for the accuracy here btw. That's another matter which needs signoff.

Reply:
> I'm not entirely comfortable promoting a shortened link, piping anything to bash involves a lot of trust so I'd rather this be explicit _and_ also present a more manual way of doing it. You could update your text to point directly to `acid-test.sh` if they want to download it themselves, and then use the full URL as an option if they're comfortable: `https://github.com/nodejs/node-gyp/blob/master/acid-test.sh`

Understood that and will expand it back to the full URL although it needs to be /raw instead of /blob.

> _But_ also this test needs to be renamed, I know this makes it tedious, but this is only an issue with macOS Catalina and we need to separate this out as a concern. I suggest: `macOS_Catalina_acid_test.sh` so it shows up right next to this doc.

Sure, it perfectly makes sense 👍